KsApi/GraphAPI.swift

```diff
@@ -723,57 +724,6 @@ public enum GraphAPI {
     }
   }
 
-  /// Necessary fields for Google Pay
```

Comment:
Why aren't we supporting Google Play anymore?

Reply:
Good question, not 100% sure why, I don't think it was ever fully supported. Also _I think_ there is potential for the Stripe payment sheet to handle both Apple Pay and Google Pay and many more payment options. Just my hunch, Isa is probably more knowledgeable about that.

Reply:
Some quick historical context that we previously supported Android Pay in a limited capacity, but don't think we actually ever supported Google Pay. +1 to Mubarak that Google Pay will be a good opportunity to us post-Payment Sheet implementation.

Reply:
Got it. Thanks for providing more context around this